apps/coscrad-frontend/src/styles.scss

```diff
@@ -1,6 +1,9 @@
 body {
     font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode',
         Geneva, Verdana, sans-serif;
+    min-height: 100vh;
```

Comment:
Be sure to give the entire app a once through and make sure there are no unintended consequences of changes to global styling.